src/openms/source/ANALYSIS/QUANTITATION/IsobaricNormalizer.cpp

```diff
@@ -66,6 +66,8 @@ namespace OpenMS
          file_it != consensus_map.getColumnHeaders().end();
          ++file_it)
     {
+      // TODO I think this does not work if you are normalizing a consensus map including multiple files
```

Comment:
so this works for IsobaricAnalyzer but not for the workflow?

Reply:
I think IsobaricNormalizer was not yet used on a ConsensusMap from multiple files. I expect it to fail/yield bogus results.
I did not add normalization yet for IsobaricWorkflow. Not sure if very relevant, since most downstream tools will do it anyway and probably better than we do. I only see advantage if we can normalize based on information that will not be present anymore downstream.